lib/extract_gauge_ghost_extended.cu

```diff
@@ -82,21 +83,22 @@ namespace quda {
 
     int oddness = (parity+arg.localParity[dim])&1;
     
+    Matrix<complex<typename mapper<Float>::type>, Ncolor(length)> u;
```

Comment:
-> create a `constexpr int nColor = Ncolor(length);` above as you've done elsewhere

Reply:
Since this is a one line use is there any point?

Reply:
Duly noted, but I've noticed declarations like this have become a style convention elsewhere in the code, so I think it'd be a good change to make just for consistency.

I also think this `constexpr ...` convention is very useful for code readability.

Reply:
That said, it's not a critical change.